README.md

```diff
@@ -4,25 +4,27 @@
 
 # Create a banking chatbot with FAQ discovery, anger detection and natural language understanding
 
-In this code pattern, we will create a chatbot using Node.js and Watson Assistant. The Assistant flow will be enhanced by using Natural Language Understanding to identify entities and using Tone Analyzer to detect customer emotions. For FAQs, a call to the Discovery service will use passage retrieval to pull answers from a collection of documents.
+In this code pattern, we will create a chatbot using Node.js and Watson Assistant. The Assistant flow will be enhanced by using Natural Language Understanding to identify location entities. For FAQs, a call to the Discovery service will use passage retrieval to pull answers from a collection of documents.
```

Comment:
are we going to detect emotions with Assistant or NLU?

Reply:
Assistant. Will add that to paragraph